src/app/screens/App/shared/components/LoggedOutHeader.js

```diff
@@ -5,33 +5,30 @@ import { LinkContainer } from 'react-router-bootstrap';
 const LoggedOutHeader = () => (
     <Navbar>
         <Nav>
-            <NavItem disabled>Personal</NavItem>
-            <NavItem disabled>Open source</NavItem>
-            <NavItem disabled>Business</NavItem>
-            <NavItem disabled>Explore</NavItem>
-            <NavItem disabled>Help</NavItem>
+            <NavItem>Personal</NavItem>
+            <NavItem>Open Source</NavItem>
+            <NavItem>Business</NavItem>
+            <NavItem>Explore</NavItem>
+            <NavItem>Personal</NavItem>
```

Comment:
This should still be 'Help'.  I realize it'll change as we get to implementing those pages but ya know